cmake/MSCCLPP.cmake

```diff
@@ -63,13 +63,17 @@ if(ENABLE_MSCCLPP)
                 WORKING_DIRECTORY ${MSCCLPP_SOURCE}
             )
         endif()
-        execute_process(
-           COMMAND git apply ${CMAKE_CURRENT_SOURCE_DIR}/ext-src/cpx.patch
-           WORKING_DIRECTORY ${MSCCLPP_SOURCE}
+	execute_process(
+            COMMAND git apply -v --recount ${CMAKE_CURRENT_SOURCE_DIR}/ext-src/cpx.patch
```

Comment:
Can you remove these changes? It might cause conflict with the other mscclpp that I am working on.